plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/ui/internal/CDK32ServerWizardFragment.java

```diff
@@ -93,6 +93,14 @@ public void modifyText(ModifyEvent e) {
 		}; 
 		msHomeText.addModifyListener(msHomeModListener);
 	}	
+	
+	private String getDefaultMinishiftHome() {
+		String msHome = System.getenv(CDK32Server.ENV_MINISHIFT_HOME);
```

Comment:
According to documentation for System.getEnv(String) :
```
The returned map will never contain null keys or values.
```
that means check for null is not needed.